haskell/private/cc_wrapper.py.tpl

```diff
@@ -570,7 +570,10 @@ def link(output, libraries, rpaths, args):
         rpaths = shorten_rpaths(rpaths, libraries, output)
 
     args.extend(rpath_args(rpaths))
-    run_cc(args, exit_on_error=True)
+    # Note: `RULES_HASKELL_DOCTEST` is only set if called from doctest,
+    #       this is used to silence the linker output to not interfere with the output
+    #       from GHCi
+    run_cc(args, capture_output="RULES_HASKELL_DOCTEST" in os.environ, exit_on_error=True)
```

Comment:
Should it maybe be called `RULES_HASKELL_SILENCE_LINKER` or something like that?

Reply:
OK, :+1: I'll change that.